src/usdm-client.ts

```diff
@@ -616,6 +640,10 @@ export class USDMClient extends BaseRestClient {
     return this.getPrivate('fapi/v1/feeBurn');
   }
 
+  signTradFiPerpsContract(): Promise<{ code: number; msg: string }> {
```

Comment:
```suggestion
  signTradFiPerpsAgreement(): Promise<{ code: number; msg: string }> {
```

Just to avoid confusion with "perps contracts"